src/flex/agentApp/src/WatsonAssistantAgentAppPlugin.tsx

```diff
@@ -0,0 +1,57 @@
+/**
+ * (C) Copyright IBM Corp. 2021.
+ *
+ * Licensed under the MIT License (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * https://opensource.org/licenses/MIT
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ */
+import * as Flex from '@twilio/flex-ui';
+import { FlexPlugin } from 'flex-plugin';
+
+const PLUGIN_NAME = 'WatsonAssistantAgentAppPlugin';
+
+export default class WatsonAssistantAgentAppPlugin extends FlexPlugin {
+  constructor() {
+    super(PLUGIN_NAME);
+  }
+
+  /**
+   * This code is run when your plugin is being started
+   * Use this to modify any UI components or attach to the actions framework
+   *
+   * @param flex { typeof Flex }
+   * @param manager { Flex.Manager }
+   */
+  init(flex: typeof Flex, manager: Flex.Manager) {
+    this.registerReducers(manager);
+
+    flex.CRMContainer.defaultProps.uriCallback = (task: any) => {
+      // Pull the session history from the task attributes
+      if (task?.attributes?.sessionHistoryKey) {
+        let { sessionHistoryKey } = task.attributes;
+        return `https://web-chat.global.assistant.watson.cloud.ibm.com/loadAgentAppFrame.html?session_history_key=${sessionHistoryKey}`;
+      }
+
+      return '';
+    }
+  }
+
+  /**
+   * Registers the plugin reducers
+   *
+   * @param manager { Flex.Manager }
+   */
+  private registerReducers(manager: Flex.Manager) {
+    if (!manager.store.addReducer) {
+      // tslint: disable-next-line
```

Comment:
What is this for? I don't see any errors on the next line. And we're not even using tslint.

Reply:
Removed this function, leftover from the code generation